src/hotspot/share/logging/logSelectionList.cpp

```diff
@@ -53,6 +53,14 @@ bool LogSelectionList::verify_selections(outputStream* out) const {
   return valid;
 }
 
+LogDecorators LogSelectionList::get_default_decorators() const {
+  for (size_t i = 0; i < _nselections; ++i) {
+    if (!LogDecorators::has_disabled_default_decorators(_selections[i])) {
+      return LogDecorators();
+    }
+  }
+  return LogDecorators(0);
```

Comment:
Here I'd like to see either an explanation of 0 as `LogDecorators(0 /* comment */)` or a meaningful name.

Reply:
There is `LogDecorators::None`

Reply:
LogDecorators::None is defined in the .cpp, so I'd either have to make it "visible" or use the alternative NoDecorators. Both options are fine for me

Reply:
I am using `LogDecorators::None` now, I think it is cleaner than the `NoDecorators` alternative